src/screens/Exchange/Exchange/Exchange.js

```diff
@@ -66,6 +66,7 @@ import FromAssetSelector from './FromAssetSelector';
 import ToAssetSelector from './ToAssetSelector';
 import OfferCard from './OfferCard';
 import { useFromAssets, useToAssets, useOffersQuery, sortOffers } from './utils';
+import { hitSlop50w20h } from '../../../utils/common';
```

Comment:
Please import it in under Utils

```suggestion
// Utils
import { hitSlop50w20h } from 'utils/common';
```

Reply:
Ah my apologies, I used the auto import function in visual studio code